test/core/pipeline/test_draw.py

```diff
@@ -111,6 +111,7 @@ def test_to_mermaid_text_does_not_edit_graph():
     assert expected_pipe == pipe.to_dict()
 
 
+@pytest.mark.skip(reason="This is a nice to have, but frequently fails due to mermaid.ink issues")
```

Comment:
I'm not sure if we should remove this test (instead of just skipping it).

This test failed multiple times in the last few days, blocking PRs.

I believe this test often fails because it calls the mermaid.ink server multiple times in a short period of time.

Reply:
I agree with this - but we could nevertheless investigate how to make this test run , let's say at least once a week.

Reply:
probably there are also other tests that we can run, e.g.: once a week, and not for every single PR if the PR doesn't affect the code related to the PR